src/shogun/machine/GeneralizedLinearMachine.cpp

```diff
@@ -0,0 +1,299 @@
+/*
+ * STILL A WORK IN PROGRESS
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Tej Sukhatme
+ */
+
+// #include <rxcpp/rx-lite.hpp>
+// #include <shogun/features/DotFeatures.h>
+// #include <shogun/labels/Labels.h>
+// #include <shogun/labels/RegressionLabels.h>
+#include <shogun/machine/GeneralizedLinearMachine.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+// #include <utility>
+#include <math.h>
```

Comment:
never...

Reply:
Should I replace it with `#include <cmath>` instead?